addon/webextension/background/communication.js

```diff
@@ -18,6 +18,7 @@ window.communication = (function () {
     let func = registeredFunctions[req.funcName];
     let result;
     try {
+      req.args.push(sender);
```

Comment:
I found I needed the `tab.id` in the `closeSelector` handler. It's probably generally useful to make this available, and I didn't find a case where it broke anything.